packages/playground/website/vite.config.ts

```diff
@@ -27,16 +27,19 @@ import { listAssetsRequiredForOfflineMode } from '../../vite-extensions/vite-lis
 // eslint-disable-next-line @nx/enforce-module-boundaries
 import virtualModule from '../../vite-extensions/vite-virtual-module';
 
-const proxy: CommonServerOptions['proxy'] = {
-	'^/plugin-proxy': {
-		target: 'https://playground.wordpress.net',
-		changeOrigin: true,
-		secure: true,
-	},
-};
-
 const path = (filename: string) => new URL(filename, import.meta.url).pathname;
 export default defineConfig(({ command, mode }) => {
+	const proxy: CommonServerOptions['proxy'] = {
+		'^/plugin-proxy': {
+			target:
+				mode === 'production'
+					? 'https://playground.wordpress.net'
+					: `http://${remoteDevServerHost}:${remoteDevServerPort}`,
```

Comment:
IIRC, the purpose of this bit is to forward requests to `playground.wordpress.net/plugin-proxy.php` when a plugin-proxy.php request is received by the the dev server. The `plugin-proxy.php` script requires PHP and a GH_TOKEN environment var, so I think maybe we want to keep forwarding such requests to playground.wordpress.net where there is both a PHP server and that secret defined.